sources/packages/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -27,28 +32,47 @@ export class AssessmentAESTController extends BaseController {
     private readonly studentAppealService: StudentAppealService,
     private readonly studentAssessmentService: StudentAssessmentService,
     private readonly assessmentControllerService: AssessmentControllerService,
+    private readonly applicationExceptionService: ApplicationExceptionService,
   ) {
     super();
   }
 
   /**
-   * Method to get all requested assessments for a student
+   * Get all requests related to an assessments for a student
    * application, i.e, this will fetch all pending and denied
-   * student appeal for an application.
-   * @param applicationId, application number.
-   * @returns assessment requests for a student application.
+   * student appeals for an application or possible application
+   * exceptions that will prevent the assessment to proceed till
+   * they are approved, for instance, when a document is uploaded
+   * and need to be reviewed.
+   * @param applicationId application number.
+   * @returns assessment requests or exceptions for a student application.
    */
   @Get("application/:applicationId/requests")
   async getRequestedAssessmentSummary(
     @Param("applicationId") applicationId: number,
   ): Promise<RequestAssessmentSummaryAPIOutDTO[]> {
+    const applicationExceptions =
+      await this.applicationExceptionService.getExceptionsByApplicationId(
+        applicationId,
+        ApplicationExceptionStatus.Pending,
+        ApplicationExceptionStatus.Declined,
+      );
+    if (applicationExceptions) {
+      return applicationExceptions.map((applicationException) => ({
+        id: applicationException.id,
+        submittedDate: applicationException.createdAt,
+        status: applicationException.exceptionStatus,
+        requestType: RequestAssessmentTypeAPIOutDTO.StudentException,
+      }));
+    }
+
     const studentAppeal =
       await this.studentAppealService.getPendingAndDeniedAppeals(applicationId);
     return studentAppeal.map((appeals) => ({
       id: appeals.id,
       submittedDate: appeals.submittedDate,
       status: appeals.status,
-      triggerType: AssessmentTriggerType.StudentAppeal,
+      requestType: RequestAssessmentTypeAPIOutDTO.StudentException,
```

Comment:
I didn't get this part. I can sync up to understand.

Reply:
Before the items displayed in the "Requests" area were mainly from some sort of record that would be triggering an assessment/reassessment request, which is no longer true because we will be also displaying an exception if any. The `requestType` provide more accurate information about what that particular object represents because the `triggerType` is actually only valid for assessment/reassessment, which is not the case here.

Reply:
Thanks for explaining over the call 👍